CHANGELOG.md

```diff
@@ -10,6 +10,66 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 9.9.0
+
+### Important Changes
+
+- **feat(browser): Add `previous_trace` span links ([#15569](https://github.com/getsentry/sentry-javascript/pull/15569))**
+
+  The `@sentry/browser` SDK and SDKs based on `@sentry/browser` now emits a link from the first root span of a newly started trace to the root span of a previously started trace. You can control this feature via an option in `browserTracingIntegration()`:
+
+  ```js
+  Sentry.init({
+    dsn: 'your-dsn-here'
+    integrations: [
+      Sentry.browserTracingIntegration({
+        // Available settings:
+        // - 'in-memory' (default): Stores previous trace information in memory
+        // - 'session-storage': Stores previous trace information in the browser's `sessionStorage`
+        // - 'off': Disable storing and sending previous trace information
+        linkPreviousTrace: 'in-memory',
+      }),
+    ],
+  });
+  ```
+
+- **feat(browser): Add `logger.X` methods to browser SDK ([#15763](https://github.com/getsentry/sentry-javascript/pull/15763))**
+
+  For Sentry's [upcoming logging product](https://github.com/getsentry/sentry/discussions/86804), the SDK now supports sending logs via dedicated
+
+  ```js
+  Sentry.init({
+    dsn: 'your-dsn-here',
+    _experiments: {
+      enableLogs: true, // This is required to use the logging features
+    },
+  });
+
+  Sentry.logger.info('This is a trace message', { userId: 123 });
+  // See PR for better documentation
+  ```
+
+  Please note that the logs product is still in early access. See the link above for more information.
+
+- **feat(nextjs): Support `instrumentation-client.ts` ([#15705](https://github.com/getsentry/sentry-javascript/pull/15705))**
```

Comment:
let's swap this and the linked traces entries here, IMHO that should be the ordering (as linked traces do not do anything for users in UI just yet 😅 )?